src/Form/index.tsx

```diff
@@ -672,8 +673,12 @@ function Form({
         .then(([session, steps]) => {
           updateBackNavMap(session.back_nav_map);
           setIntegrations(session.integrations);
-          if (!isObjectEmpty(initialValues))
-            clientInstance.submitCustom(initialValues, false);
+          if (!isObjectEmpty(initialValues)) {
+            clientInstance.submitCustom(initialValues, {
+              override: false,
+              servarKeyToTypeMap: getServarTypeMap(steps)
```

Comment:
could you just cast initialValues before passing it into submitCustom? might be a cleaner API